css/style.css

```
@@ -16,4 +16,129 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* This is Header (Navigation & Hero )*/
+
+/* Navigation */
+nav{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:140px;
+    width: 80%;
+    margin: auto;
+    font-size: 2.5rem;
```

```suggestion
    font-size: 1rem;
```
All the text on this page is quite big! We could reduce the values of all font-sizes in this file